website/components/Image.tsx

```diff
@@ -90,6 +90,25 @@ export const getOptimizedMediaUrl = (opts: OptimizationOptions) => {
     return originalSrc;
   }
 
+  if (
+    isImageOptmizationEnabled() ||
+    originalSrc.startsWith(
+      "https://ozksgdmyrqcxcwhnbepg.supabase.co/storage",
+    ) ||
+    originalSrc.startsWith(
+      "https://deco-sites-assets.s3.sa-east-1.amazonaws.com/",
+    )
```

Comment:
wdyt about this snippet:

```tsx
const ALLOWED_ORIGINS = ["https://ozksgdmyrqcxcwhnbepg.supabase.co/storage",  "https://deco-sites-assets.s3.sa-east-1.amazonaws.com/"]
```

```suggestion
    ALLOWED_ORIGINS.some(origin => originalSrc.startsWith(origin))
```